src/custom-elements.js

```diff
@@ -697,9 +728,19 @@ let Deferred;
    */
   function _createElement(doc, tagName, options, callConstructor) {
     const customElements = _customElements();
+    let isAttr;
+    if (options && options.is) {
+        // We're going to take care of setting the is attribute ourselves
+        isAttr = options.is;
+        delete options.is;
+    }
     const element = options ? _nativeCreateElement.call(doc, tagName, options) :
       _nativeCreateElement.call(doc, tagName);
-    const definition = customElements._definitions.get(tagName.toLowerCase());
+    if (isAttr) {
+        element.setAttribute('is', isAttr);
+        element.is = isAttr;
```

Comment:
Good catch -- I think I confused the ["is value"](https://dom.spec.whatwg.org/#concept-element-is-value) with a property on the dom element. But it seems like the "is value" is just an internal thing not exposed to userland.